src/Bounds.cpp

```diff
@@ -1063,6 +1083,8 @@ class BoxesTouched : public IRGraphVisitor {
     bool consider_calls, consider_provides;
     Scope<Interval> scope;
     const FuncValueBounds &func_bounds;
+    Scope<Expr> let_stmts;
+    map<string, set<string>> children; // All vars which values depend on var in 'key'
```

Comment:
This comment is a little too terse for me to unpack. How about:

Maps from each variable in scope to the variables whose definitions depend on it.